qcodes/utils/helpers.py

```diff
@@ -352,6 +352,9 @@ def __exit__(self, type, value, tb):
             self.logger.addHandler(handler)
 
 
+@deprecate(
+    reason=("Instrument names are cross process identifiers that should be "
```

Comment:
to me the deprecation reason is specific for the use of the function, but in principle this function could've been used not only in station. but that's not important at all :)

Reply:
This is very true, agreed!